yaraneba/app/controllers/application_controller.rb

```diff
@@ -18,4 +18,12 @@ class Forbidden < ActionController::ActionControllerError; end
   private def rescue404()
     render 'errors/not_found', status: 404
   end
+
+  private def login_check
+    redirect_to login_path if session[:user_id].blank?
+  end
+
+  private def not_logged_in_check
+    redirect_to tasks_path if session[:user_id].present?
+  end
```

Comment:
```suggestion
  private
  
  def login_check
    redirect_to login_path if session[:user_id].blank?
  end

  def not_logged_in_check
    redirect_to tasks_path if session[:user_id].present?
  end
```

private は 1箇所でまとめれます